pkg/api/user_token_test.go

```diff
@@ -0,0 +1,298 @@
+package api
```

Comment:
Not super happy with these tests in combination with admin_user_test. Happy with the test of the internal methods, but testing routes to make sure that correct user id is used; user id from context or user id from route.